js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -99,6 +99,14 @@ const DropDownEditor = TextBox.inherit({
         });
     },
 
+    _isInstantlyMode() {
```

Comment:
we really need have this method(for one usage)?

Reply:
For one? [[1](https://github.com/DevExpress/DevExtreme/blob/a1e7d621115b5534efcaf73aca216e0c36fc7bb8/js/ui/date_box/ui.date_box.base.js#L66), [2](https://github.com/DevExpress/DevExtreme/blob/a1e7d621115b5534efcaf73aca216e0c36fc7bb8/js/ui/drop_down_editor/ui.drop_down_editor.js#L107)].

Reply:
missed second place